hw2/task01/task01.py

```diff
@@ -0,0 +1,80 @@
+"""
+Given a file containing text.
+
+Complete using only default collections:
+    1) Find 10 longest words consisting from largest amount of unique symbols
+    2) Find rarest symbol for document
+    3) Count every punctuation char
+    4) Count every non ascii char
+    5) Find most common non ascii char for document
+"""
+
+import string
+from collections import Counter
+
+
+def open_file_and_count_chars(file_path: str) -> Counter:
+    """Open file and decipher and count chars."""
+    counter = Counter()
+    with open(file_path, "r", encoding="utf-8") as text:
+        for line in text:
+            while "\\u" in line:
```

Comment:
There is another way to deal with this text.

Reply:
`find` will be used. That will reduce the number of operations.

Reply:
Am a talking about a completely different approach.  You need to avoid parsing these values manually.